scripts/sysman/power.yml

```diff
@@ -97,6 +97,8 @@ members:
     - type: int32_t
       name: "maxLimit"
       desc: "[out] (Deprecated) The maximum power limit in milliwatts that can be requested. A value of -1 means that this is not known."
+details:
+    - "1.7": "Note: This structure is deprecated and functionality is provided by $s_power_new_properties_t."
```

Comment:
`- "1.7": "[DEPRECATED] Use $s_power_new_properties_t."`

Reply:
removing this note entirely here since we actually don't have the new struct defined. shall add new struct first and then go through deprecation process